waba_integration/whatsapp_business_api_integration/workspace/whatsapp/whatsapp.json

```diff
@@ -1,32 +1,29 @@
 {
- "cards_label": "Reports & Masters",
- "category": "Modules",
  "charts": [],
- "charts_label": "",
- "creation": "2022-09-27 17:41:47.382370",
- "developer_mode_only": 0,
- "disable_user_customization": 0,
+ "content": "[{\"id\":\"wc3HHqsg-p\",\"type\":\"shortcut\",\"data\":{\"shortcut_name\":\"WABA Settings\",\"col\":3}},{\"id\":\"Y_1bDAzZbc\",\"type\":\"card\",\"data\":{\"card_name\":\"Communication\",\"col\":4}},{\"id\":\"9HACpwJXJq\",\"type\":\"card\",\"data\":{\"card_name\":\"Process Tracking\",\"col\":4}}]",
```

Comment:
This seems to be the new standard in frappe 14+. Not sure if `id` is required / indicated, though, this is autogenerated.